components/Notifications/NotificationsList.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react'
+import Notification from './Notification'
+
+export default function NotificationsList() {
+  return (
+    <>
+      <Notification />
+    </>
+  )
+}
```

Comment:
this should receive a list of notification objects as props

Reply:
and use a for loop to create multiple `<Notification />` objects and pass each notification object as props to `<Notification />`